src/components/TagInput/TagInput.props.ts

```diff
@@ -0,0 +1,16 @@
+import { InputHTMLAttributes } from 'react';
+
+import type { TagInputConfig } from './TagInput.styles';
+
+import { BasicInputState } from '@/types';
+
+export type TagInputProps = {
+  custom?: TagInputConfig;
+  state?: BasicInputState;
+  isValidationError?: boolean;
+  label?: string;
+  helperText?: string;
```

Comment:
Same story ;) 

<img width="192" alt="image" src="https://github.com/VirtusLab/tetrisly-react/assets/6291752/3b940db6-ebae-4848-a689-996689736cc9">

Reply:
Should the user decide whether there is a `BeforeIcon` and what `Intent` is? Or should it be included in `TagInput` logic and rendered based on that?
I ask because I assumed icon only appears when there is alert or validation error and now I'm not sure if it's correct

Reply:
changed